### PR TITLE
Add services section

### LINKED ISSUE #51: Adding a Service section
## 🫤 Description

<br>

> This section will be containing all types of services that  portfolio owner will be giving. 

<br>

## 📍 Use Case

<br>

> It can be used to display the work such as Web Development ,UI/UX Designer , SEO Expert.

<br>

## ✋🏻 Upvotes

<br>

> If you support this feature request, add a 👍🏻.

<br>

## 🌱 Contribute

<br>

> - [x] I want to contribute 

  <br>

## 🎉 Hacktoberfest

<br>

> - [X] I want to participate 

<br>

## 📌 Prototype

<br>

Screenshot from 2023-10-03 14-46-10

### PR DESCRIPTION
**Pull request by Kawaljeet-943**
# Add Services Section to Minifolio

## Short Description
Added a services section to the portfolio page to showcase the different services offered.

## Points Added Or Removed
- Added a new services section with brief descriptions of each service offered.

## Links To Issues
- Closes #51

## Repository Title with Link
[Minifolio](https://github.com/Kawaljeet-943/Minifolio)

## Files Changed
- `index.html` - Added the HTML structure for the services section.
- `css/style.css` - Added styles for the newly created services section.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -572,6 +572,88 @@ a:visited {
...
+}
+
+.serviceText {
+  text-align: center;
+}
+
+.serviceText .text {
+  font-weight: bold;
+  font-size: 20px;
+  color: #272341;
```
Comment (divyanshudhruv) [inline css/style.css:648]: You can add - font-family: "Montserrat" !important;

**Comment by divyanshudhruv** [comment]
@Kawaljeet-943 that looks cool. But there are some styling problems, I hope you can fix it. Here is the fixed code, you can replace the old piece of code with this 
 one and customize it or keep it simple. Good luck

### CODE

```css
.serviceText {
  text-align: center;
  font-family: "Poppins" !important;
  font-size: 15px;

}

.serviceText .text {
  font-weight: bold;
  font-size: 20px;
  color: #272341;
  font-family: "Poppins" !important;
  margin-bottom: 10px;

}

.serviceText .textMain {
  font-size: 16px;
  color: #555;
  font-family: "Montserrat" !important;

}

```

**Comment by Kawaljeet-943** [comment]
Thank you so much for the feedback, @divyanshudhruv I really appreciate the suggestions. I'll replace the old code with the one you've provided and make any necessary customizations. I'll let you know once it's updated. Thanks again for helping me improve the styling!

**Comment by Kawaljeet-943** [comment]
I've made the suggested changes to the style in the services section. Please have a look and let me know if there's anything else that needs adjustment. Looking forward to your feedback!

**Comment by divyanshudhruv** [comment]
sure, I will review it soon

**Comment by divyanshudhruv** [comment]
@Kawaljeet-943 that seems fine to me. Merging it soon. Good luck

**Review by divyanshudhruv — APPROVED** [review]
Thank you for the contribution. It looks awesome

**Pull request merged**